packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -3887,6 +3887,22 @@ describe('plugin-meetings', () => {
               assert.isRejected((Promise.reject()));
             }
           });
+
+          it('updates remote mute state when brb is enabled', async () => {
+            meeting.audio = { handleServerRemoteMuteUpdate: sinon.stub() };
+
+            await meeting.beRightBack(true);
+
+            sinon.assert.calledOnceWithExactly(meeting.audio.handleServerRemoteMuteUpdate, meeting, true);
+          });
+
+          it('does not update remote mute state when brb is disabled', async () => {
+            meeting.audio = { handleServerRemoteMuteUpdate: sinon.stub() };
+
+            await meeting.beRightBack(false);
+
+            assert.isTrue(meeting.audio.handleServerRemoteMuteUpdate.notCalled);
```

Comment:
nitpick: you could just do `assert.notCalled(meeting.audio.handleServerRemoteMuteUpdate);` - feels more readable